src/OpenLoco/MessageManager.cpp

```diff
@@ -37,9 +40,74 @@ namespace OpenLoco::MessageManager
         call(0x004285BA, regs);
     }
 
+    // 0x00428E0F
+    static void clearActiveMessage()
+    {
+        if (getGameState().activeMessageIndex != MessageId::null)
+        {
+            get(getGameState().activeMessageIndex)->var_C8 = 0xFFFF;
```

Comment:
any clue what `var_C8` would be?

Reply:
That is what I've been working on for a while. I think it's the time it's been active